### DISCUSSION TITLE
families with the largest difference in plant

### DISCUSSION THREAD

**Discussion by yywyaoyaowu** [hahnlab/CAFE5#83] [Q&A]
Hi 
I tried to run cafe5, and I get the error: When I remove these DOGs, I still get the same error but with different OG ID. it takes forever to remove these OGs. I work on 100 plant genomes. Thanks very much!
Families with largest size differentials:
OG0000110: 99
OG0000201: 99
OG0000065: 99
OG0005545: 99
OG0000118: 98
OG0000054: 98
OG0000125: 98
OG0000310: 97
OG0000359: 97
OG0000018: 97
OG0000206: 96
OG0000658: 96
OG0000138: 96
OG0000685: 96
OG0000132: 95
OG0000429: 95
OG0007094: 95
OG0000326: 94
OG0016297: 93
OG0000277: 93

You may want to try removing the top few families with the largest difference
between the max and min counts and then re-run the analysis.

**Comment by benfulton** [comment]
You should just be able to remove those families from your input family file. I'm not seeing an error in your message - was CAFE unable to find a lambda value?
